### DISCUSSION TITLE
Newbie VSCode question: An orangish bar covers a line of code making it unreadable

### DISCUSSION THREAD

**Discussion by marceepoonu** [microsoft/vscode-discussions#1074] [Extension Development QnA] [answered]
I suspect this is the wrong forum, and I hope that someone will be able to give me some suggestions about how to find the right Forum to pose my question. I am a newbie trying to run some python code in visual studio code, and I think a debugger or something like that is obscuring the python code, making it impossible for me to see the code and fix the error. Any suggestions about where to find out what is producing the orangish bar would be much appreciated, and how to adjust the color would be much appreciated.
<img width="1575" alt="2024-02-27_VSC_orangish_bar_covers_txt_making_it_unreadable" src="https://github.com/microsoft/vscode-discussions/assets/529625/48b00ff2-21b4-4a35-9e22-88aef78c6848">

**Comment by alefragnani** [comment, marked as answer]
Hi @marceepoonu ,

Yes, you are correct about this not being the right forum for this question, as this forum is intended for extension development. But don't worry, let's try to point you in the right direction.

First of all, if you are having an issue with some extension, the best place to look for help would be in the extension's page. About Python, if you are using the Microsoft extension (https://marketplace.visualstudio.com/items?itemName=ms-python.python) they suggest you to ask questions in Stack Overflow (https://stackoverflow.com/questions/tagged/visual-studio-code+python). I don't know about the Python extension, but I'm not sure that _orangish bar_ comes from it. 

I would, instead, disable each extension individually, up until the _orangish bar_ disappears. Doing so, you could easily identify which extension is actually proving that bars.

Maybe this bars comes from some git or debug extension, or control chars that your theme is not rendering properly. Unfortunately, I couldn't identify anything based on the screenshot. Sorry.

Hope this helps

**Reply by marceepoonu** [reply]
Thank you.  Much appreciated!
